XiEditor/Search.swift

```diff
@@ -149,7 +149,10 @@ extension EditViewController {
         if term != nil {
             params["chars"] = term
         }
-        
+
+        if term ?? "" == "" {
```

Comment:
This seems very confusing to me. Perhaps if you wanted to check for empty strings, you could use `term.isEmpty`?

Reply:
Ah yea I'm trying to check if the string is nil _or_ the empty string. So maybe `(term ?? "").isEmpty` would be more clear.